tools/nui/FileReader.cpp

```diff
@@ -10,7 +9,9 @@ FileReader::FileReader(const QString& file_) : file(file_) {
 void FileReader::process() {
   timer.start();
   // TODO: Support reading files from the API
```

Comment:
Remove TODO

Reply:
Right now, code only supports using public api, should I add a check to use either use old method or public api?